app/controllers/fhir/r4/api_controller.rb

```diff
@@ -4,12 +4,10 @@
 class Fhir::R4::ApiController < ActionController::API
   include ActionController::MimeResponds
   before_action only: %i[create update] do
-    doorkeeper_authorize! :'user/*.write', :'user/*.*'
-    check_api_access
+    doorkeeper_authorize! :'user/*.write', :'user/*.*', :'system/*.write', :'system/*.*'
```

Comment:
Reduce the scope to what API can do. 'user/Patient.write' and 'system/Patient.write'